.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,24 +11,16 @@ body:
 
         Prerequisites
 
-        - Rust >= 1.75
         - Node.js >= 18.18.0
-        - Yarn >= 4
```

Comment:
`rust` and `yarn` is not necessary for a user who use `rolldown`,  I am not sure what is the lowest nodejs version do we support?